configs/recommended.js

```diff
@@ -0,0 +1,95 @@
+const typescriptRules = {
+  'no-unused-vars': 'off',
+  '@typescript-eslint/no-unused-vars': 'off',
+  '@typescript-eslint/no-unused-vars-experimental': ['error', { ignoreArgsIfArgsAfterAreUsed: true }],
+  '@typescript-eslint/no-var-requires': 'off',
+  '@typescript-eslint/no-explicit-any': 'off',
+  '@typescript-eslint/no-use-before-define': 'off',
+  '@typescript-eslint/ban-ts-comment': 'off',
+  'semi': 'off',
+  '@typescript-eslint/semi': 'error',
+  '@typescript-eslint/type-annotation-spacing': 'error',
+  '@typescript-eslint/member-delimiter-style': 'error',
+  '@typescript-eslint/indent': ['error', 2],
+  '@typescript-eslint/explicit-member-accessibility': ['error', { accessibility: 'no-public' }],
```

Comment:
(Opinion, optional)
In Java, methods are private by default (which is a very good approach). Explicitly typing out 'public' might force developers to think about the scope of a method/variable and maybe reduce the public API, inadvertently helping anyone to comes across that class (as a user of that class). As opposed to defining all the member methods without any regard to property accessors.

Alternatively, this may be considered hand-holding or simply annoying to some developers.

Reply:
Both perspectives are valid :thinking: @Blakko @szydan @fabiocorneti what do ye think about forcing explicit access modifiers?

Reply:
I think it makes sense to force it, don't see this as a particularly annoying thing.

Reply:
Then it shall be so